modules/atomic-widgets/props-handler/transformers-registry.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Elementor\Modules\AtomicWidgets\PropsHandler;
+
+use Elementor\Core\Utils\Collection;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Transformers_Registry extends Collection {
+	public function register( Transformer $transformer ): self {
+		$this->items[ $transformer->get_type() ] = $transformer;
+
+		return $this;
+	}
```

Comment:
should we do any validation or throw an exception when the type is already registered?